packages/node/src/generators/application/files/app/src/main.ts__tmpl__

```diff
@@ -1 +1,13 @@
-console.log('Hello World!');
+<% if(framework === 'express') {%>
```

Comment:
I think it'd be easier to put different framework files in their own directory.

```
files/express/src/main.ts__tmpl__
files/connect/src/main.ts__tmpl__
files/fastify/src/main.ts__tmpl__
files/koa/src/main.ts__tmpl__
```

We can rename existing `files/app` to `files/common` and generate with that + the framework specific file that overrides the main.ts file